test/ApplyArgs.purs

```diff
@@ -0,0 +1,170 @@
+module Test.Ctl.ApplyArgs (suite, main, contract) where
+
+import Contract.Prelude
+
+import Aeson (Aeson, encodeAeson)
+import Contract.Config (ServerConfig, testnetConfig)
+import Contract.Monad (Contract, launchAff_, runContract, throwContractError)
+import Contract.PlutusData (PlutusData(List, Map, Bytes), toData)
+import Contract.Prim.ByteArray (byteArrayToHex, hexToByteArrayUnsafe)
+import Contract.Scripts (PlutusScript(PlutusScript), Validator(Validator))
+import Contract.Transaction (Language, plutusV1Script)
+import Ctl.Examples.AlwaysSucceeds (alwaysSucceedsScript)
+import Ctl.Examples.IncludeDatum (only42Script)
+import Ctl.Examples.PlutusV2.Scripts.AlwaysSucceeds (alwaysSucceedsScriptV2)
+import Ctl.Internal.ApplyArgs (applyArgs)
+import Ctl.Internal.Helpers ((<</>>))
+import Ctl.Internal.Plutip.Server (stopChildProcessWithPort)
+import Ctl.Internal.Plutip.Spawn
+  ( ManagedProcess
+  , NewOutputAction(NoOp, Success)
+  , spawn
+  )
+import Ctl.Internal.QueryM
+  ( ClientError(ClientEncodingError)
+  , handleAffjaxResponse
+  , mkHttpUrl
+  , postAeson
+  , scriptToAeson
+  )
+import Ctl.Internal.Serialization (toBytes) as Serialization
+import Ctl.Internal.Serialization.PlutusData (convertPlutusData) as Serialization
+import Ctl.Internal.Test.TestPlanM (TestPlanM, interpret)
+import Data.BigInt (fromInt)
+import Data.List.Lazy (replicate)
+import Data.String (Pattern(Pattern))
+import Data.String as String
+import Data.UInt (UInt)
+import Data.UInt as UInt
+import Effect.Aff (bracket)
+import Foreign.Object as Object
+import Mote (group, test)
+import Node.ChildProcess (defaultSpawnOptions)
+import Untagged.Union (asOneOf)
+
+contract ∷ Contract () Unit
+contract = liftEffect main
+
+contractApply ∷ Array PlutusData → Contract () Validator → Contract () Unit
```

Comment:
```suggestion
contractApply ∷ Array PlutusData -> Contract () Validator -> Contract () Unit
```
I don't know if this change is really needed, maybe only for consistency?

Reply:
Let's not use unicode symbols

Reply:
silly IDE